source/extensions/filters/network/dubbo_proxy/hessian_utils.cc

```diff
@@ -24,6 +24,54 @@ typename std::enable_if<std::is_signed<T>::value, T>::type leftShift(T left, uin
   return left << bit_number;
 }
 
+inline void addByte(Buffer::Instance& buffer, const uint8_t value) { buffer.add(&value, 1); }
+
+void addSeq(Buffer::Instance& buffer, const std::initializer_list<uint8_t>& values) {
+  for (const int8_t& value : values) {
+    buffer.add(&value, 1);
+  }
+}
+
+size_t doWriteString(Buffer::Instance& instance, const absl::string_view& str_view) {
```

Comment:
nit: string_view by value

Reply:
I have modified it to: size_t doWriteString(Buffer::Instance& instance, absl::string_view str_view).